tests/python/gaia-ui-tests/gaiatest/tests/functional/lockscreen/test_lockscreen_unlock_to_camera_with_passcode.py

```diff
@@ -17,7 +17,8 @@ def setUp(self):
         # Turn off geolocation prompt
         self.apps.set_permission('Camera', 'geolocation', 'deny')
 
-        self.data_layer.set_setting('lockscreen.passcode-lock.code', self._input_passcode)
+        self.setPasscodeTo1337()
```

Comment:
Although I'm worry about it looks like you fix the magic number on the function with its name, rather than write a generic function that could accept any passcode like `setPasscodeTo(1337)`, but for test I think it's OK. Of course if it's possible to change the code to `setPasscodeTo(...)` is better.